content/authors/alex-measure/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+# View this page at https://digital.gov/authors/alex-measure
+# Learn how to edit our pages at https://workflow.digital.gov
+
+# slug — the specific user-id for an author.
+slug: alex-measure
+display_name: "Alex Measure"
+first_name: "Alex"
+last_name: "Measure"
+
+# List your pronoun(s) if you want them displayed alongside your name. If blank, we'll use just your name. Learn more http://mypronouns.org
+pronoun: ""
+
+# Email — If you include an email address, it will be displayed on your profile page
+email: 
+
+# Bio — keep it under 50 words
+bio: "Alex Measure is an economist-turned-machine learning and natural language processing (NLP) practitioner at BLS. He designs, builds, and maintains machine learning systems that automate difficult text classification, information extraction, and record-matching problems in production systems."
```

Comment:
Can you spell out BLS here since it's not the same as the agency listing @DanielJPino-ux